### PR TITLE
OSDOCS-3693: Generic ephemeral vols correction

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.11+

Issue: [OSDOCS-3693](https://issues.redhat.com/browse/OSDOCS-3693)

Link to docs preview: http://file.rdu.redhat.com/lpettyjo/OSDOCS-3693-correction/storage/generic-ephemeral-vols.html

Additional information: This is a correction to remove content regarding default SCC issue.

PTAL: @jsafrane, @Phaow, @gcharot, @gnufied

### CODE REVIEW

**Comment by Phaow** [comment]
Hi @lpettyjo ,Looks good to me. My bad, thanks for help update!

**Review by sayjadha — COMMENTED** [review]
LGTM.

File: storage/generic-ephemeral-vols.adoc [hunk 1]
```diff
@@ -19,11 +19,4 @@ include::modules/storage-ephemeral-vols-security.adoc[leveloffset=+1]
 
 include::modules/storage-ephemeral-vols-pvc-naming.adoc[leveloffset=+1]
 
-include::modules/storage-ephemeral-vols-scc-bug.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../authentication/managing-security-context-constraints.adoc[Managing security context constraints]
```
Comment (gnufied) [inline storage/generic-ephemeral-vols.adoc:27]: Don't we need to actually remove the bug doc from repo?

**Comment by gnufied** [comment]
Are you thinking of adding release-notes/known-issues in same PR or different PR?

**Comment by lpettyjo** [comment]
> Are you thinking of adding release-notes/known-issues in same PR or different PR?

There will be a separate PR for this. @JoeAldinger is pursuing this.

**Comment by gcharot** [comment]
lgtm

**Pull request merged**